modules/social_features/social_page/social_page.install

```diff
@@ -263,7 +263,7 @@ function social_page_update_111101(): void {
  * Permissions to check:
  * - "translate page node".
  */
-function social_page_update_111102(): void {
```

Comment:
This change should land on 11.x as it it BC. Hook update number should not be changed.